backend/orchestrator/common_pb2_grpc.pyi

```diff
@@ -2,3 +2,16 @@
 @generated by mypy-protobuf.  Do not edit manually!
 isort:skip_file
 """
+import abc
+import collections.abc
+import grpc
+import grpc.aio
```

Comment:
Did we change the version of protobuf used or something?

Reply:
🤷‍♂️  no idea